pkg/command/service/generate.go

```diff
@@ -48,8 +48,8 @@ func NewServiceGenerateCommand(p *pkg.PerfParams) *cobra.Command {
 		Short: "generate Knative Service",
 		Long: `generate Knative Service workload
 For example:
-# To generate Knative Service workload
-kperf service generate —n 500 —-interval 20 —-batch 20 --min-scale 0 --max-scale 5 (--namespace-prefix testns/ --namespace nsname)
+# To generate ksvc workload
```

Comment:
change from ksvc to Knative Service in this commit https://github.com/knative-sandbox/kperf/commit/3442681383f775d8a7a5d6e30e4c18e14e4ed5e9#diff-9ae03d0addd0e6cdae57d9ca422eb11e2f67a7437c82b23caf9128aa0cf4bc94,
why set it back?

Reply:
Fixed, and also fixed in clean.go `To clean Knative Service workload` and measure.go `To measure a Knative Service creation time running currently with 20 concurent jobs`